src/Uno.UI.Runtime.Skia.Wpf/WpfHost.cs

```diff
@@ -307,8 +298,6 @@ private void WpfHost_SizeChanged(object sender, SizeChangedEventArgs e)
 			);
 		}
 
-		public SKSize CanvasSize => bitmap == null ? SKSize.Empty : new SKSize(bitmap.PixelWidth, bitmap.PixelHeight);
```

Comment:
Is a public API change intended here?

Reply:
I removed it because it's not used anywhere in uno repository. If it were used by someone there would already be an issue because at the moment it always returns an empty rectangle.

Reply:
@workgroupengineering This didn't use to be the case before https://github.com/unoplatform/uno/pull/8612 😕 

`bitmap` was previously getting assigned in `OnRender`
`designMode` used to be used too.

cc @MartinZikmund

Reply:
after #8612
bitmap is manage from UnoWpfRenderer
is assigned only and never uses other parts.

Reply:
@workgroupengineering I mean if someone was using the `CanvasSize` API, then #8612 should have broken them. So instead of removing the field and `CanvasStyle`, I think the correct action might be returning the `bitmap` from `UnoWpfRenderer.Render` so that we keep `CanvasSize` functioning as before.

Reply:
Or probably move `CanvasSize` to `UnoWpfRenderer` and call it from `WpfHost`

Reply:
If we want to avoid breaking change here, we can do it like suggested and add the actual removal as planned breaking change for next major release